app/components/pilas-blockly.js

```diff
@@ -431,7 +433,7 @@ export default Component.extend({
   },
 
   shouldShowCongratulationsModal() {
-    return this.experiments.isNotAffected()
+    return this.experiments.shouldShowCongratulationsModal(this.challenge)
```

Comment:
Nono, el congratulations modal es sólo para not affected. Ver mis otros comments :)